src/components/pagination/index.tsx

```diff
@@ -48,18 +48,16 @@ const Pagination = ({
 			'Pagination: pageSize is required, but was not specified. Please try passing a non-zero, positive value such as `10`.'
 		)
 	}
-	if (_page < 1) {
+	if (page < 1) {
 		throw new Error(
 			'Pagination: page must be a non-zero, positive number. Please try passing a value such as `1`.'
 		)
 	}
 
-	const [page, _setPage] = useState(_page)
```

Comment:
This component now has no internal `page` state, and must be controlled externally, via props (which should be reasonable since we're already passing that value in)